### PR TITLE
Change order of contributors to right order

### LINKED ISSUE #661: Fix contributor order
While fixing the branch for #507 the order of contributors wasn't considered. In the branch WIP-637 it's right. See for example https://github.com/hbz/lobid-resources/blob/WIP-639-addLabelToComplexSubjects/src/test/resources/jsonld/CT003012479

### PR DESCRIPTION
**Pull request by ChristophEwertowski**
Resolves #661

### CODE REVIEW

**Comment by acka47** [comment]
+1 Looks like you finally made, @ChristophEwertowski & @dr0i . Congrats and thanks for the persistence.

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 1]
```diff
@@ -3520,17 +3572,55 @@
 			<data source="@contributionBnode[12][0123456789][02468]"/>
 			<data source="@relatorNode[12][0123456789][02468]"/>
 		</combine>
+		<!-- tried: @contributionBnode100|@contributionBnode200 -->
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: these comments are now save to be removed

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 2]
```diff
@@ -3520,17 +3572,55 @@
...
+		</entity> -->
+		
+		
 		<entity name="~rdf:list" reset="true">
 			<combine name="http://id.loc.gov/ontologies/bibframe/contribution" value="${a}"
 				sameEntity="true" reset="true">
 				<data source="@contributionBnode[12][0123456789][02468]" name="a"/>
 				<data source="@relatorNode[12][0123456789][02468]"/>
-			</combine>
+				<!-- <data source="@contributionBnode1[0123456789][02468]|@contributionBnode2[0123456789][02468]" name="a"/>
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: these comments are now save to be removed

File: src/main/resources/morph-hbz01-to-lobid.xml [hunk 3]
```diff
@@ -3520,17 +3572,55 @@
...
 		</entity>
+	
+	
+	
+		
 		<combine name="~rdf:subject" value="${a}">
 			<data source="@id"/>
 			<data source="@contributionBnode[12][0123456789][02468]" name="a"/>
 			<data source="@relatorNode[12][0123456789][02468]"/>
+			<!-- <data source="@contributionBnode" name="a"/>
```
Comment (dr0i) [inline src/main/resources/morph-hbz01-to-lobid.xml]: these comments are now save to be removed

**Pull request merged**